homeassistant/components/androidtv/media_player.py

```diff
@@ -491,6 +495,10 @@ def unique_id(self):
         """Return the device unique id."""
         return self._unique_id
 
+    async def async_close(self, event):
```

Comment:
We don't need this method on the entity. We can access `aftv` outside the entity.

Reply:
Like this? https://github.com/home-assistant/core/commit/fa83875ddc286b59fcc20c9f67c63102e11c86bc